src/components/InputBox.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './InputBox.css';
+
+interface InputBoxProps {
+  value: string;
+  type?: string;
+  id: string;
+  legend: string;
+  isRequired?: boolean;
+  onChange: ({ target }: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const InputBox = ({
+  value,
+  type = 'text',
+  id,
+  legend,
+  isRequired = true,
+  onChange,
+}: InputBoxProps) => {
+  return (
+    <fieldset className="InputBox">
+      <legend className="InputBox__legend">
+        {legend}
+        <span className="InputBox__star">{isRequired && `*`}</span>
```

Comment:
```suggestion
        {isRequired && <span className="InputBox__star">*</span>}
```
아예 엘리먼트를 감싸지 않은 이유가 있을까욥

Reply:
너무 되는대로 짰더니 코드가 많이 지저분하네요.. 딱히 이유는 없습니다. 다음 커밋에 수정하도록 하겠습니다!